olm-catalog/serverless-operator/1.6.0/serverless-operator.v1.6.0.clusterserviceversion.yaml

```diff
@@ -91,35 +91,96 @@ spec:
+      and SaaS when needed.
 
-    Knative Serving builds on Kubernetes to support deploying and
-    serving of serverless applications and functions. Serving is easy
-    to get started with and scales to support advanced scenarios. The
-    Knative Serving project provides middleware primitives that
-    enable:
+    ## Components & APIs
+
+    This operator provides the following components:
+
+    ### Knative Serving
+
+    Knative Serving builds on Kubernetes to support deploying and serving of
+    applications and functions as serverless containers. Serving is easy to get
+    started with and scales to support advanced scenarios. Other features
+    includes: 
 
     - Rapid deployment of serverless containers
     - Automatic scaling up and down to zero
     - Routing and network programming
     - Point-in-time snapshots of deployed code and configurations
 
-    ## Prerequisites
+    ![](https://i.imgur.com/vqL48B8.png)
 
-    The Serverless Operator's provided APIs such as Knative Serving
-    have certain requirements with regards to the size of the underlying
-    cluster. See [Getting started with OpenShift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/serverless_applications/index#serverless-getting-started)
-    for more info.
+
+    ### Knative Client - `kn`
+
+    The Knative client `kn` is your door to the Knative world. It allows you to
+    create Knative resources interactively from the command line or from within
+    Shell scripts.
+
+    `kn` offers you:
+
+    - Full support for managing all features of Knative Serving (services,
+      revisions, traffic splits) 
+    - Growing support Knative eventing, closely following its development
+      (managing of sources & triggers) 
+    - A plugin architecture similar to that of kubectl plugins 
+    - A thin client-specific API in golang which helps in tasks like synchronously
+      waiting on Knative service write operations.  
+    - An easy integration of Knative into Tekton Pipelines by using kn in a Tekton
+      Task.
+
+    <!-- ### Knative Eventing
+    Knative Eventing offers a consistent, Kubernetes-based solution for reliable,
+    secure and scalable asynchronous event delivery from packaged or app-created
+    event sources to on-cluster and off-cluster consumers.
+
+
+    -->
 
     ## Further Information
 
-    For documentation on using Knative Serving, see the
-    [serving section](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/serverless_applications/index#knative-serving).
+    For documentation on OpenShift Serverless, see the [installation
+    guide](https://docs.openshift.com/container-platform/4.3/serverless/installing-openshift-serverless.html)
+    or the [getting
+    started](https://docs.openshift.com/container-platform/4.3/serverless/serverless-getting-started.html)
+    sections.
```

Comment:
I don't think we need to include both, it just increases the chance of broken links and adds more maintenance. I would just use the getting started one.